docs/en/stack/data-frames/checkpoints.asciidoc

```diff
@@ -0,0 +1,60 @@
+[role="xpack"]
+[[ml-transform-checkpoints]]
+=== {dataframe-transform-cap} checkpoints
+++++
+<titleabbrev>Checkpoints</titleabbrev>
+++++
+
+beta[]
+
+Each time a {dataframe-transform} examines the source indices and creates or
+updates the destination index, it generates a _checkpoint_.
+
+If your {dataframe-transform} runs only once, there is logically only one
+checkpoint. If your {dataframe-transform} runs continuously, however, it creates
+checkpoints as it ingests and transforms new source data.
+
+For example, a {cdataframe-transform}:
```

Comment:
These are not really "examples" .. these are steps performed during checkpointing.